content/docs/command-reference/push.md

```diff
@@ -18,55 +18,44 @@ positional arguments:
 ## Description
 
 The `dvc pull` and `dvc push` commands are the means for uploading and
-downloading data to and from remote storage. These commands are similar to
-`git pull` and `git push`, respectively (with some key differences given the
-nature of DVC, see details below).
+downloading data to and from remote storage (S3, SSH, GCS, etc.). These commands
+are similar to `git pull` and `git push`, respectively.
 
 [Data sharing](/doc/use-cases/sharing-data-and-model-files) across environments,
 and preserving data versions (input datasets, intermediate results, models,
-[metrics](/doc/command-reference/metrics), etc.)
-[remotely](/doc/command-reference/remote) are the two most common use cases for
-these commands.
+[metrics](/doc/command-reference/metrics), etc.) remotely are the most common
+use cases for these commands.
 
-The `dvc push` command allows us to upload data to remote storage. It doesn't
-save any changes to the code, `dvc.yaml`, or `.dvc` files (those should be saved
-with `git commit` and `git push`).
+The `dvc push` command uploads data to
+[remote storage](/doc/command-reference/remote). It doesn't save any changes to
+the code, `dvc.yaml`, or `.dvc` files (that should be saved with `git commit`
+and `git push`).
 
-💡 For convenience, a Git hook is available to automate running `dvc push` after
-`git push`. See `dvc install` for more details.
-
-Under the hood a few actions are taken:
+The default remote is used (see `dvc config core.remote`) unless the `--remote`
+option is used. See `dvc remote` for more information on how to configure a
+remote.
 
-- The push command by default uses all stages (in `dvc.yaml` and `dvc.lock`) and
-  `.dvc` files in the <abbr>workspace</abbr>. The command options will either
-  limit or expand the set of stages or `.dvc` files to consult.
+Without arguments, it uploads all files and directories missing from remote
+storage, found as <abbr>outputs</abbr> in the stages (in `dvc.lock`) or `.dvc`
```

Comment:
dvc.yaml?

Reply:
Hmmm yeah... Kinda. dvc.lock is where you find the output file names — that's what this meant. But probably this paragraph should be simplified and not even mention either one. See 246f446c.